static/app/views/onboarding/welcome.tsx

```diff
@@ -74,6 +75,11 @@ class OnboardingWelcome extends Component<Props> {
                 onClick={() => {
                   triggerFall();
                   onComplete({});
+                  trackAdvancedAnalyticsEvent(
+                    'growth.onboarding_im_ready',
```

Comment:
@roggenkemper Actually, instead of recording an event for this click, can we record an event for the select platform page loading? It will make it easier to set up the A/B test later.